packages/vendor-core/package.json

```diff
@@ -26,6 +26,7 @@
   ],
   "dependencies": {
     "@novnc/novnc": "^1.0.0",
+    "@patternfly/react-icons": "^3.14.2",
```

Comment:
Any reason to add this new dep? looks like a leftover

Reply:
@sharvit This PR is just so I could test your other PR; it's not meant to be merged. :smile:

Reply:
Haha thanks!